lua/mapvote/cl_mapvote.lua

```diff
@@ -34,15 +33,15 @@ net.Receive( "MapVote_VoteStarted", function()
         object["map"] = map
         object["playcount"] = playCount
 
-        MapVote.CurrentMaps[#MapVote.CurrentMaps + 1] = object
+        MapVote.currentMaps[#MapVote.currentMaps + 1] = object
```

Comment:
```suggestion
        table.insert( MapVote.currentMaps, object )
```

Reply:
im going to leave this for now, because I plan on rewriting alot of the old cl code anwyay.